websites/B/bol.com/dist/metadata.json

```diff
@@ -0,0 +1,27 @@
+{
+  "author": {
+    "name": "QkeleQ10",
+    "id": "807917674477649943"
+  },
+  "service": "bol.com",
+  "description": {
+    "en": "Bol.com is a webshop in the Netherlands and Belgium offering over 20 million products.",
+    "nl": "Bol.com is een webwinkel in Nederland en België met meer dan 20 miljoen artikelen."
+  },
+  "url": "www.bol.com",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/EFsXguy.png",
```

Comment:
Fix logo, its blurry and the dot is half outside of the image....

Reply:
Dot thing is intentional, that's just how the logo is. Upscaled the image.